products/bigquery/api.yaml

```diff
@@ -0,0 +1,144 @@
+# Copyright 2017 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+--- !ruby/object:Api::Product
+name: Google Cloud BigQuery
+prefix: gbigquery
+versions:
+  - !ruby/object:Api::Product::Version
+    name: v2
+    base_url: https://www.googleapis.com/bigquery/v2/
+    default: 
+scopes:
+  - https://www.googleapis.com/auth/bigquery
+objects:
+  - !ruby/object:Api::Resource
+    name: 'Dataset'
+    kind: 'bigquery#dataset'
+    base_url: projects/{{project}}/datasets
+    description: >
```

Comment:
Don't we usually use `|`? What does `>` do?

Reply:
Ooh, I learned this one yesterday!  There are 63 types of block quote in yaml.  `>` vs `|` is one of the distinctions.  'flow' vs 'folded' - `>` means that only blank lines count as newlines in the resulting text.  So:

```
thing: |
  foo bar
  baz

  qux
```
is
`foo bar\nbaz\n\nqux`, but if it's `thing: >` instead, it's `foo bar baz\nqux`.

Reply:
I've been wanting to bring this up and keep forgetting.

I think we should be using `>` instead of `|` for basically everything.

(Also, you think Nathan is joking when he says 63, but the real number is pretty close to that. YAML is so weird.)

Reply:
I am not!  :)

2 block styles, each with 2 possible block chomping indicators (or none), and with 9 possible indentation indicators (or none), 1 plain style and 2 quoted styles.

2 x (2 + 1) x (9 + 1) + 1 + 2 = 63

Reply:
Never again will I assume you're using hyperbole

Reply:
I still don't know what an indentation indicator is!  I don't think they interact too heavily with the rest of the options, though - nobody online is complaining about them, at least.

Anyway, back to the original point, you're right, `>` is better than `|` for making pretty markdown and docs.  Let's switch.